bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -973,94 +968,141 @@ impl EcAddOp {
         assert_eq!(output.len(), 64);
 
         let mut buf = [0u8; 32];
-        let point_p = g1_from_slice(&mut buf, &input[0x00..0x40]).unwrap();
-        let point_q = g1_from_slice(&mut buf, &input[0x40..0x80]).unwrap();
-        let point_r_got = g1_from_slice(&mut buf, &output[0x00..0x40]).unwrap();
-        assert_eq!(G1Affine::from(point_p.add(&point_q)), point_r_got);
+        let opt_point_p: Option<G1Affine> = g1_from_slice(&mut buf, &input[0x00..0x40]).into();
+        let opt_point_q: Option<G1Affine> = g1_from_slice(&mut buf, &input[0x40..0x80]).into();
+        let point_r_evm = g1_from_slice(&mut buf, &output[0x00..0x40]).unwrap();
```

Comment:
todo(zhuo): check this unwrap

Reply:
since the output bytes are from EVM, they will produce `[0u8; 64]` or a point on curve G1. Hence the `unwrap`

Reply:
I see